frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -190,7 +190,13 @@ export const menuActions = [
   Kebab.factory.Delete,
 ];
 
-export const menuActionsCreator = (kindObj: K8sKind, vm: VMKind, { vmi, pods, migrations }) => {
+type ExtraResources = { vmi: VMIKind; pods: K8sResourceKind[]; migrations: K8sResourceKind[] };
```

Comment:
Fixing types to work with [improved `KebabOptionsCreator`](https://github.com/openshift/console/pull/2106/files#diff-45aa3dfc05205bd3d1b3da31bd999e6dR101).